tests/display/Container.tests.ts

```diff
@@ -0,0 +1,937 @@
+// import { AlphaFilter } from '../../src/filters/defaults/alpha/AlphaFilter';
+// import { Rectangle } from '../../src/maths/shapes/Rectangle';
+import { Container } from '../../src/scene/container/Container';
+// import { Graphics } from '../../src/scene/graphics/shared/Graphics';
+import { getRenderer } from '../utils/getRenderer';
+
+import type { WebGLRenderer } from '../../src/rendering/renderers/gl/WebGLRenderer';
+
+// function testAddChild(fn: any)
+// {
+//     return () =>
+//     {
```

Comment:
any reason for keeping around all this commented code?

Reply:
Not at all, thanks for spotting!